### PR TITLE
Fix/lacks in documentation

### PR DESCRIPTION
**Pull request by paweljozwicki**

### CODE REVIEW

File: documentation/diagrams/src/websight-e2e-testing-ci.puml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    node docker_image as "WS Docker Image"
+  }
+
+  rectangle e2d_step as "Step 2. Run E2E tests" {
+    rectangle browser as "Browser" {
+      node cypress_iframe as "Cypress Tests (iFrame)"
+      node ws_iframe as "Application (iFrame)"
+    }
+  }
+  rectangle bb_service as "Docker Service" {
```
Comment (tomaszmichalak) [inline documentation/diagrams/src/websight-e2e-testing-ci.puml:20]: This is not valid. BB requires additional `service` to run a docker container. In GitHub Actions Docker is available on the runner host.

File: tests/end-to-end/README.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ Resource: `tests/authentication.spec.ts`
 
 ## How to run in CI?
```
Comment (tomaszmichalak) [inline tests/end-to-end/README.md:77]: Can we provide details what do we want to present? I think all CI details can be skipped.

File: tests/end-to-end/README.md [hunk 3]
```diff
@@ -32,7 +32,7 @@ to run all tests in the background.
 
 The architecture for running Cypress locally looks like below:
```
Comment (tomaszmichalak) [inline tests/end-to-end/README.md:33]: Can we provide details what do we want to present? Please validate this diagram with: https://blog.codecentric.de/en/2020/10/cypress-ui-end2end-testing/.

File: tests/end-to-end/README.md (hunk 2 above)
Comment (paweljozwicki) [reply tests/end-to-end/README.md]: I just moved some documentation from Howlite, but I think there was too much unnecessary informations. I have to remove them from Howlite as well.

**Pull request merged**